way_to_home/custom_user/views.py

```diff
@@ -116,3 +125,68 @@ def signin_google(request):
         return HttpResponse("User was successfully created", status=201)
 
     return HttpResponse("User's data is empty", status=400)
+
+
+@require_http_methods(["POST"])
+def reset_password(request):
+    """Function that provides reset user password"""
+    data = request.body
+    email = data.get('email')
+    if not email:
+        return RESPONSE_400_OBJECT_NOT_FOUND
+    user = CustomUser.get_by_email(email=email)
+    if not user:
+        return RESPONSE_400_OBJECT_NOT_FOUND
+    token = create_token(data={'email': user.email})
+    send_email_password_update(user, token)
+    return RESPONSE_200_OK
+
+
+@require_http_methods(['PUT'])
+def confirm_reset_password(request, token):
+    """Function that provides confirm reset user password"""
+    data = request.body
+    new_password = data.get('new_password')
+    confirm = decode_token(token)
+
+    if not confirm:
+        return RESPONSE_498_INVALID_TOKEN
+    user = CustomUser.get_by_email(email=confirm.get('email'))
+
+    if not user or not new_password:
+        return RESPONSE_400_OBJECT_NOT_FOUND
+
+    if not password_validator(new_password):
+        return RESPONSE_400_INVALID_DATA
```

Comment:
```suggestion
    if not user or not new_password:
        return RESPONSE_400_OBJECT_NOT_FOUND

    if not password_validator(new_password):
        return RESPONSE_400_INVALID_DATA
```
I think it is not correct to return `object_not_found` if `new_password` is `None`. So you can remove this check `not new_password`.

This case will be handled by `password_validator`. `password_validator` returns `False` if value is `None`.

Reply:
changed